quantum/template/ps2avrgb/config.h

```diff
@@ -43,8 +43,6 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define BACKLIGHT_LEVELS 1
 #define RGBLIGHT_ANIMATIONS
 
-#define NO_UART 1
```

Comment:
Sorry, this should be reverted, as it does appear to be important.

Reply:
Oh really? How so? What does it affect?

Reply:
https://github.com/qmk/qmk_firmware/blob/ee700b2e831067bdb7584425569b61bc6329247b/tmk_core/protocol/vusb.mk#L12-L17

And it will cause compile errors, if it's not handled correctly (which I've seen already)

Reply:
got it!